compiler/circle-input-names/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+file(GLOB SOURCES "src/*.cpp")
+
+add_executable(circle_input_names ${SOURCES})
+target_include_directories(circle_input_names PUBLIC include)
```

Comment:
I do not see `include` folder (in also draft). can we remove this line?

Reply:
Oh, I'll remove it.
Thanks ;)